### ISSUE TITLE
Remove "Basic Authentication (Deprecated)" and "Basic Authentication With API Token"

### ISSUE THREAD

**Issue by diver80** [floralvikings/jira-connector#199]
While starting with jira-connector it would be help if non working (anymore) authentication mechanisms would be removed

- Basic Authentication (Deprecated)
- Basic Authentication With API Token

At least for me only base64 is working.

**Comment by MrRefactoring** [comment]
At the moment, these mechanisms are in demand. There are old Jira servers that are not updated. They may well work with these features.

**Comment by diver80** [comment]
Yes you are right - on server they are still working. Would be great to add ad comment about not working against Atlassian cloud ;-). I'll change issue summary - thanks for quick reply.

**Comment by MrRefactoring** [comment]
It would be cool if you showed PR.  I am ready to watch it

**Comment by stbumu** [comment]
would OAuth work with jira server?
